tests/conftest.py

```diff
@@ -0,0 +1,25 @@
+"""Global pytest fixtures"""
+
+import pytest
+
+
+#
+# requests data
+#
+@pytest.fixture()
+def mock_data():
```

Comment:
that's a very generic fixture name, maybe `mock_request_kwargs`? It looks like something that you could pass to `m.get(**mock_request_kwargs)`